terraform/cloudfoundry/acl_cell_local_in.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_network_acl_rule" "90_local_in" {
+    network_acl_id = "${aws_network_acl.cell.id}"
+    protocol = -1
+    rule_number = 90
+    rule_action = "allow"
+    cidr_block = "10.0.0.0/16"
```

Comment:
Ditto the comment in https://github.com/alphagov/paas-cf/pull/102/files#r52454687 - we can use https://github.com/alphagov/paas-cf/blob/7a95a56745577c580ca189ac380d528da17ed82a/terraform/globals.tf#L14-L17